backend-tests/tests/test_rbac_v2.py

```diff
@@ -471,19 +588,32 @@ def test_deploy_to_group(self, clean_mongo, test_case):
                         "permission_sets_with_scope": [
                             {
                                 "name": "DeployToDevices",
-                                "scope": {"type": "DeviceGroups", "value": ["test"],},
-                            },
-                            {"name": "ManageTokens",},
+                                "scope": {"type": "DeviceGroups", "value": ["test"]},
+                            }
                         ],
-                    },
+                    }
                 ],
                 "device_groups": {"test": 1, "production": 1},
-                "deploy_group": "test",
-                "set_configuration_status_code": 204,
-                "deploy_configuration_status_code": 200,
+                "deploy_group": "production",
```

Comment:
this way you test the scope, not the permission;
to test the permission, the deploy_group should be "test"

Reply:
Thanks, done!